src/components/style/forms/checkbox.css

```diff
@@ -0,0 +1,40 @@
+/* stylelint-disable selector-max-combinators */
+/* stylelint-disable selector-max-compound-selectors */
```

Comment:
Lol cheating.... (jk)

Reply:
I would if it worked. I've resorted to doing this until I have time to further investigate why line-specific comments are ignored by the linter